app/helpers/user_journey_helper.rb

```diff
@@ -33,7 +33,7 @@ def display_component(component)
     elsif component.vsp
       COMPONENT_TYPE::VSP_SHORT
     else
-      COMPONENT_TYPE::SP_SHORT
+      COMPONENT_TYPE::SP_LONG
```

Comment:
do we need to change this?

Reply:
In the prototype the service provider is never referred to as just 'SP', instead its 'service provider'

Reply:
should this line then become `t 'user_journey.component_name.SP'` and no further changes would be required? there's the existing 'conversion' from `SP` to `Service Provider`

Reply:
But that existing one is capitalised,  dont know how much of an issue that is? The constant follows the same flow as the other components when finding the component name in the tests and views. Makes the sp feel like part of the family rather than being unique when reading the code

Reply:
I could just the capitalise method when need be, which would only be on two occurrences?

Reply:
I think that'd be a better approach then changing the yaml files and locales

Reply:
Sorry to just pick up where we left from, it would be better to use the constant `SP_LONG = 'service provider'.freeze` and just capitalise it when need be. Rather than changing the yaml files?

Reply:
yep!